files/socialite/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "Socialite",
+  "repo": "tmort/Socialite",
+  "files": {
+    "include": ["socialite.js>", "socialite.min.js", "extensions/*.js"]
```

Comment:
This will do what you want:
`"include": ["socialite.js", "socialite.min.js", "./extensions/*.js"]`